synthetic_phi/whole_slide_image/src/isyntax.py

```diff
@@ -0,0 +1,159 @@
+import os
+import traceback
+
+from lxml import etree as ET
+import base64
+import io
+from PIL import Image, ImageDraw
+
+def generate_image(x, y, image_text1=None, image_text2=None):
+    img = Image.new('RGB', (x, y), color=(73, 109, 137))
```

Comment:
(73, 109, 137) are these random numbers or hold any significance? Can you make a constant with a name so we know what these numbers are for?

Reply:
an example: https://github.com/project-medid/project-medid/blob/main/synthetic_phi/ultrasound/src/utils.py#L80

Reply:
Good idea, thank you. No special significance. I added a named constant DEFAULT_BACKGROUND_BLUE.